pubnub/endpoints/objects_v2/channel/get_all_channels.py

```diff
@@ -1,17 +1,18 @@
 from pubnub.endpoints.objects_v2.objects_endpoint import ObjectsEndpoint, ListEndpoint, \
-    IncludeCustomEndpoint
+    IncludeCustomEndpoint, IncludeStatusTypeEndpoint
```

Comment:
Is `IncludeStatusTypeEndpoint` interface common for include type and status?

Reply:
Yeah. I've packed them into one class but you can manage them separately. I think when they always appear together there's no sense in separating them.